rewrite-python/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id("org.openrewrite.build.language-library")
+}
+
+dependencies {
+    api(project(":rewrite-core"))
+    api("org.jetbrains:annotations:latest.release")
+    api("com.fasterxml.jackson.core:jackson-annotations")
+
+    implementation(files("lib/intellij-python-plugin.jar"))
```

Comment:
too large to include in the commit

Reply:
Do they publish this to any repository? We could always fat-jar it in to rewrite-python as necessary